EncoreTickets.SDK/Utilities/Common/Serializers/SingleOrListJsonSerializer.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace EncoreTickets.SDK.Utilities.Common.Serializers
+{
+    class SingleOrListJsonSerializer<T> : DefaultJsonSerializer
+    {
+        protected override Newtonsoft.Json.JsonSerializer Serializer
+        {
+            get
+            {
+                var serializer = base.Serializer;
+                serializer.Converters.Add(new SingleOrListConverter<T>());
+                return serializer;
```

Comment:
Please remove this logic from the getter, otherwise a new converter will be added every time someone tries to get a serializer